SPECS/rpm/rpm.spec

```diff
@@ -32,8 +32,6 @@ Requires:       debugedit
 Requires:       libarchive
 Requires:       libselinux
 Requires:       lua
-Requires:       rpm-libs = %{version}-%{release}
```

Comment:
Are we removing this requirement because this packages doesn't need the libraries, or because the dependency is already exposed through the shared library requirements auto-generated by RPM ?

Reply:
This was more of a test to see what's pulling all of the -devel subpackages into the core images. I don't think this particular requires should actually be removed